source/Sanbox.JobRunner/Sandbox.JobRunner.csproj

```diff
@@ -1,60 +1,24 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProjectGuid>{C75CE4B3-EF65-45B8-A1D2-E5D5CA6F1157}</ProjectGuid>
     <OutputType>Exe</OutputType>
-    <RootNamespace>Sandbox.JobRunner</RootNamespace>
-    <AssemblyName>Sandbox.JobRunner</AssemblyName>
-    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
+    <TargetFramework>net472</TargetFramework>

```

Comment:
In that case, check here as well 😉

Reply:
These are just "executable" samples which don't necessary have to be upgrade.
But I can upgrade them to netcoreapp2.0